.eslintrc.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": ["airbnb-base", "prettier"],
   "plugins": ["prettier", "jest"],
+  "parserOptions": {
+    "ecmaVersion": 11
```

Comment:
needed for optional chaining (babel transpiles it away)

Reply:
thinking about it, #125 _might_ be considered breaking as well, if people previously ignored an engine warning and used this package on node 6. I don't consider it a breaking change, but bundling these together in a new major removes any ambiguity 🤷